pipeline/tasks/publish_tasks.py

```diff
@@ -56,3 +58,19 @@ def execute(self, repo_url, username, password, publish_env,
 
     def validate(self):
         return []
+
+
+class GitHubPushTask(task_base.TaskBase):
+    """Uploads local files to GitHub repository in a new commit. Won't
```

Comment:
Nit: prefer to start docstrings with a single summary line, and give details in the following paragraph.